identity-iota/src/tangle/client_builder.rs

```diff
@@ -47,6 +49,11 @@ impl ClientBuilder {
     self
   }
 
+  pub fn retry_until_included(mut self, value: bool) -> Self {
```

Comment:
Missing doc-comment.

```suggestion
  /// When publishing to the Tangle, sets whether to retry until the message is 
  /// confirmed by a milestone.
  /// Default: true.
  pub fn retry_until_included(mut self, value: bool) -> Self {
```